<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCommandMoveObject</class>
 <widget class="QDialog" name="DialogCommandMoveObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Move object...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
       <item>
        <widget class="QLabel" name="labelObjectID">
         <property name="text">
          <string>Object ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PanelPrimitiveValue" name="widgetPrimitiveObjectID" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>208</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="WidgetSuperTree" name="treeView">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxIgnore">
           <property name="text">
            <string>Ignore if impossible</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxWaitEnd">
           <property name="text">
            <string>Wait the end of moves</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxCameraOrientation">
           <property name="text">
            <string>With camera orientation</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxMoves">
         <property name="title">
          <string>Moves</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="labelStepSquareMoves">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>17</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>17</height>
                 </size>
                </property>
                <property name="text">
                 <string>Step / square moves:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxStepSquare"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStepSquareNorth">
                <property name="text">
                 <string>1 to North</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStepSquareSouth">
                <property name="text">
                 <string>1 to South</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStepSquareWest">
                <property name="text">
                 <string>1 to West</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStepSquareEast">
                <property name="text">
                 <string>1 to East</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStepSquareNorthWest">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>1 to North-West</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStepSquareNorthEast">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>1 to North-East</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStepSquareSouthWest">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>1 to South-West</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStepSquareSouthEast">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>1 to South-East</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStepSquareRandom">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>1 to random</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStepSquareHero">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>1 to the hero</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStepSquareOppositeHero">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>1 opposite to the hero</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStepSquareFront">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>1 in front</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStepSquareBack">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>1 back</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0,0,0,0,0,0,0,1,0,0,0,0">
              <item>
               <widget class="QLabel" name="labelChangeDirection">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>17</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>17</height>
                 </size>
                </property>
                <property name="text">
                 <string>Change direction:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonEmpty">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_25">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Turn to North</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_19">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Turn to South</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_20">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Turn to West</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_16">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Turn to East</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_21">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Turn to North-West</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_17">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Turn to North-East</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_15">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Turn to South-West</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_24">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Turn to South-East</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_18">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Turn 45° right</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_14">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Turn 45° left</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_37">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Turn 180*</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_13">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Look at the hero</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_22">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Look at the hero opposite</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
              <item>
               <widget class="QLabel" name="labelChangeObjectProperties">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>17</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>17</height>
                 </size>
                </property>
                <property name="text">
                 <string>Change object properties:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_2">
                <item>
                 <property name="text">
                  <string>ON</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>OFF</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_26">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Change speed...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_35">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Change frequency...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_31">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Move animation</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_33">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Stop animation</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_34">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Fix direction</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_36">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Through</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_23">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Set with camera</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_28">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Change graphics...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_27">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Change opacity...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_29">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Change bending...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_32">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_30">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>....</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetSuperTree</class>
   <extends>QTreeView</extends>
   <header>widgetsupertree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCommandMoveObject</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCommandMoveObject</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
